File: solutions/1528-kids-with-the-greatest-number-of-candies/solution.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        n=max(candies)
        return [True  if i+extraCandies>=n else False for i in candies]

File: solutions/1528-kids-with-the-greatest-number-of-candies/solution.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int n=0;
        List<Boolean>ls=new ArrayList<>();
        for(int i:candies)
            if(i>n)
                n=i;
        for(int i:candies)
        {
            if((i+extraCandies)>=n)
                ls.add(true);
            else
                ls.add(false);
        }
        return ls;
    }
}
